### ISSUE TITLE
Socket list includes undefined -> undefined:undefined

### ISSUE THREAD

**Issue by gajus** [myndzi/wtfnode#3]
```
(node) _linklist module is deprecated. Please use a userland alternative.
[WTF Node?] open handles:
- Sockets:
  - undefined:undefined -> undefined:undefined
    - Listeners:
  - undefined:undefined -> undefined:undefined
    - Listeners:
```

Expected output: Should not include list of undefined sockets.

Node v0.5.10.

**Comment by myndzi** [comment]
IIRC this is stdin and stdout, I thought I published a fix but maybe not. I'll put it on my task list :)

**Comment by myndzi** [comment]
Fixed and published @ 0.2.2
